core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.directive.html

```diff
@@ -1,5 +1,5 @@
 <div class="oppia-translator-overview">
-  <div class="oppia-language-selector-section">
+  <div>
```

Comment:
Maybe remove this `<div>` if it's not being used for any class?

Reply:
It's used as a container for the .oppia-translator-overview flex. (and inside it there are two divs one for language-selector and another for mode switch.)